public/js/admin.js

```diff
@@ -143,7 +143,7 @@ var displayQuestionForm = function() {
             $('#qType').change(function(evt) {
                 // get the answer part for the form requested
                 // id to replace is qAnswer
-                var form = "/" + $(this).val();
+                var form = $(this).val();
```

Comment:
why are we using var here, can we use const?

Reply:
sure